byoeb-v1/byoeb/byoeb/apis/user.py

```diff
@@ -39,4 +43,25 @@ async def get_users(request: Request):
     return JSONResponse(
         content=response.message,
         status_code=response.status_code
-    )
\ No newline at end of file
+    )
+
+def user_mcps_router(mcp):
+    class UserInput(BaseModel):
+        name: str = Field(..., min_length=3, max_length=100)
+        language: Literal["en", "hi", "mr", "te"] = Field(..., description="Supported language codes")
+        state: str = Field(..., description="Name of a state in India")
+
+    @mcp.tool
+    async def asha_register_user(data: UserInput):
+        """
+        Register a new Asha user.
+        """
+        response = await dependency_setup.users_handler.aregister([dict(
+            user_id=str(random.randint(10000, 99999)),
```

Comment:
Instead of getting random number we can get id from uuid
```suggestion
            from uuid import uuid4
            user_id=str(uuid4())
```

Reply:
Ah, I just looked at [user_svc.aregister](https://github.com/A4i-tech/byoeb/blob/7c9b61d9afce18c31981104bed16d346015a114a/byoeb-v1/byoeb/byoeb/services/user/user.py#L241), user_id="" here should work as well as aregister() rebuilds a User object with a generated user_id anyway.

Reply:
Replaced with `user_id=get_user_ids_from_phone_number_ids([phone_number])[0]`